CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 cmake_minimum_required(VERSION 3.16.3)
 project(Elastix)
 
+if(NOT CMAKE_CXX_STANDARD)
```

Comment:
`NOT` -> `NOT DEFINED`

Reply:
Yes, I suppose both work with different advantages (explicit vs works with empty string). Let's keep this, then :1234: